Tool/src/packaging/sign-packages.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+echo "****************************************"
+echo "GPG signing for daemon assets"
+echo "****************************************"
+
+echo "Starting gpg signing for zip files"
+for filename in build/dist/*; do
```

Comment:
This script will look for `build/dist` directory from the root of the package?

Reply:
Yes. From the root of the aws-xray-daemon repo checked out on GH runner.